source/math/FloatSize3D.ooc

```diff
@@ -34,8 +34,8 @@ FloatSize3D: cover {
 	init: func@ ~default { this init(0.0f, 0.0f, 0.0f) }
 	pNorm: func (p: Float) -> Float {
 		p == 1 ?
-		this width abs() + this height abs() :
-		(this width abs() pow(p) + this height abs() pow(p)) pow(1 / p)
+		this width abs() + this height abs() + this depth abs() :
+		(this width abs() pow(p) + this height abs() pow(p) + this depth abs() pow(p)) pow(1 / p)
```

Comment:
I'm a little worried `1 / p` will return an `Int`...  can you make it `1.0f / p` instead?